### PR TITLE
[testsuite] Extra test case for submitting malformed transaction

### PR DESCRIPTION
**Pull request by runtian-zhou**
## Motivation

Create a testcase for submitting transaction with wrong number of arguments.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: testsuite/tests/libratest/smoke_test.rs [hunk 1]
```diff
@@ -874,3 +874,28 @@ fn test_client_waypoints() {
         .test_validator_connection()
         .is_err());
 }
+
+#[test]
+fn test_malformed_script() {
+    let (_swarm, mut client_proxy) = setup_swarm_and_client_proxy(1, 0);
+    client_proxy.create_next_account(false).unwrap();
+    client_proxy
+        .mint_coins(&["mintb", "0", "100"], true)
```
Comment (sausagee) [inline testsuite/tests/libratest/smoke_test.rs]: Interesting, 0x0 was reserved. 

nit, mind adding a note what it test trying to catch? Like minting against reserved address.

Comment (sausagee) [reply testsuite/tests/libratest/smoke_test.rs]: LGTM otherwise

Comment (runtian-zhou) [reply testsuite/tests/libratest/smoke_test.rs]: We aren't minting to 0x0 but rather create an account in client with local index 0.

**Comment by sausagee** [comment]
@bors-libra r=sausagee

FYI please use r=handle if PR is delegated

**Pull request closed without merging**